gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -179,14 +182,14 @@ class BaselineIdea extends AbstractBaselinePlugin {
     /**
      * Extracts copyright headers from Baseline directory and adds them to Idea project XML node.
      */
-    private void addCopyright(node) {
-        def copyrightManager = node.component.find { it.'@name' == 'CopyrightManager' }
+    private void addCopyright(Node node) {
+        Node copyrightManager = node.component.find {it.'@name' == 'CopyrightManager'}
```

Comment:
a lot of these changes were motivated by `@CompileStatic`, but I couldn't fully lock it in because of all our xml manipulation. Figured it was better to just get closer 🤷